packages/contentstack-bulk-publish/package.json

```diff
@@ -1,30 +1,31 @@
 {
   "name": "@contentstack/cli-cm-bulk-publish",
   "description": "Contentstack CLI plugin for bulk publish actions",
-  "version": "1.1.0",
+  "version": "1.2.0",
   "author": "Contentstack",
   "bugs": "https://github.com/contentstack/cli/issues",
   "dependencies": {
-    "@contentstack/cli-command": "^1.1.0",
-    "@contentstack/cli-utilities": "^1.1.0",
+    "@contentstack/cli-command": "^1.2.0",
+    "@contentstack/cli-utilities": "^1.2.0",
     "@contentstack/management": "^1.6.1",
+    "@oclif/core": "^2.3.0",
```

Comment:
@aman19K If we use @oclif/core from the utility package we can remove the dependency from package.json right.?

Reply:
@antonyagustine removed @oclif/core dependency from package.json